### ISSUE TITLE
Popovers attached to system views like `NavigationView` interfere with animations

### ISSUE THREAD

**Issue by aheze** [aheze/Popovers#21]
Code:

```swift
NavigationView {
    
}
.popover(...)
```

Result:

https://user-images.githubusercontent.com/49819455/152633230-f23178c6-6faa-4e16-bb6e-cccfffcd8977.mov

**Comment by aheze** [comment]
The problem was in `WindowReader`. Before, there was a `VStack` that shoved a `WindowHandlerRepresentable` next to the view. Now it's just a single view which fixes all the animation problems.

```swift
public var body: some View {
    view(window)
        .environment(\.window, window)
        .background(
            WindowHandlerRepresentable(binding: $window)
        )
}
```

Result:

https://user-images.githubusercontent.com/49819455/152633232-e2b123c2-dfc4-4a2a-a527-99b035a9030e.mov

**Comment by aheze** [comment]
Fixed in d6aab56bdac2572b1e440e4c0781b4333945fc79. I'll release it along with some other improvements in v1.3.0, hopefully tomorrow.

**Comment by aheze** [comment]
Released in [1.3.0](https://github.com/aheze/Popovers/releases/tag/1.3.0).

**Comment by aheze** [comment]
If there were any leftover bugs, https://github.com/aheze/Popovers/releases/tag/1.3.2 probably fixes them.
